plugins/services/src/js/components/forms/EnvironmentFormSection.js

```diff
@@ -1,4 +1,5 @@
 import React, {Component} from 'react';
+import {MountService} from 'foundation-ui';
```

Comment:
currently I export Mount Service (and other services as well) like this:
```
module.exports = {
  ...
  MountService: {
    Mount,
    MountService
  },
  ...
};
```

Let's agree on how to do it. I'm ok with exporting `mount` It'd be more clear